src/Refinery/KindlyTo/Group.php

```diff
@@ -0,0 +1,190 @@
+     *   - 0 and "0", mapping to false, as well as 1 and "1", mapping to true
+     * This doesn't support:
+     *   - "null" or null, since the absence of some data is something else then
+     *     true or false
+     *   - 1.0 or 0.0, because we don't expect that someone really wants to transmit
+     *     booleans disguised as floats.
+     *
+     * All other data will be discarded. We could have decided to use a much more
+     * liberal approach by e.g. interpreting "existence of data" as true and absence
+     * of data as false. However, these transformations here are not meant to
+     * interpret incoming data as desired at all costs, but instead try to be
+     * forgiving regarding various quirks in encoding when different systems talk
+     * to each other. There seem to be some more or less sane ways to encode bools,
+     * but writing "some data" to represent true, or an empty list to represent
+     * is something else. Also, being more liberal introduces a various odd places
+     * in the mapping. If, e.g., we'd map a null to false, an empty list to false
+     * as well, would we map [null] to false or to true? Why? All this problems
+     * seem to introduce more problems than they solve, so we decided to not be
+     * very liberal here.
+     */
+    public function bool() : Transformation
+    {
+        throw new \LogicException("Not implemented yet.");
+    }
+
+    /**
+     * Get a kind transformation to a `DateTimeImmutable`.
+     *
+     * This supports:
+     *   - all formats mentioned in DateTimeInterface, which are proped in a
+     *     sensible order
+     *   - integers, which will be interpreted as Unix timestamps.
+     *
+     * All other data will be discarded.
+     */
+    public function dateTime() : DateTimeTransformation
+    {
+        throw new \LogicException("Not implemented yet.");
+    }
+
+    /**
+     * Get a kind transformation to a list.
+     *
+     * This supports all data represented as PHP array, which will be used via
+     * array_values($v). Non-arrays will be wrapped in one.
+     */
+    public function listOf(Transformation $transformation) : Transformation
+    {
+        throw new \LogicException("Not implemented yet.");
+    }
+
+    /**
+     * Get a kind transformation to a dictionary.
+     *
+     * This supports all data represented as PHP array. Non-arrays will be wrapped
+     * in one.
+     */
+    public function dictOf(Transformation $transformation) : Transformation
+    {
+        throw new \LogicException("Not implemented yet.");
+    }
+
+    /**
+     * Get a kind transformation to a tuple.
+     *
+     * This supports all data represented as PHP array, which will be used via
+     * array_values($V). Non-arrays will be wrapped in one.
+     */
+    public function tupleOf(array $transformation) : Transformation
+    {
+        throw new \LogicException("Not implemented yet.");
+    }
+
+    /**
+     * Get a kind transformation to a tuple.
```

Comment:
```suggestion
     * Get a kind transformation to a record.
```
Müsste record sein, oder?